### PR TITLE
add parse_mode to Telegram agent

### PR DESCRIPTION
**Pull request by andrey-yantsen**
With this path it will be possible to embed some html/markdown into messages

### CODE REVIEW

File: app/models/agents/telegram_agent.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@ def default_options
     def validate_options
       errors.add(:base, 'auth_token is required') unless options['auth_token'].present?
       errors.add(:base, 'chat_id is required') unless options['chat_id'].present?
+      errors.add(:base, 'parse_mode has invalid value: should be html or markdown') if options['parse_mode'].present? and ['html', 'markdown'].include? options['parse_mode']
```
Comment (cantino) [inline app/models/agents/telegram_agent.rb]: Shouldn't it be `&& !['html', 'markdown'].include? options['parse_mode']`?

File: app/models/agents/telegram_agent.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ class TelegramAgent < Agent
 
       It is assumed that events have either a `text`, `photo`, `audio`, `document` or `video` key. You can use the EventFormattingAgent if your event does not provide these keys.
 
-      The value of `text` key is sent as a plain text message.
+      The value of `text` key is sent as a plain text message. Also you can specify `parse_mode`, how Telegram will parse the message, - the value should be `html` or `markdown`.
```
Comment (cantino) [inline app/models/agents/telegram_agent.rb]: Maybe

"You can also tell Telegram how to parse the message with `parse_mode`, set to either `html` or `markdown`."

File: app/models/agents/telegram_agent.rb [hunk 3]
```diff
@@ -70,6 +71,7 @@ def receive_event(event)
 
     def send_telegram_message(method, params)
       params[:chat_id] = interpolated['chat_id']
+      params[:parse_mode] = options['parse_mode'] if options['parse_mode'].present?
```
Comment (cantino) [inline app/models/agents/telegram_agent.rb]: Consider `interpolated['parse_mode']` so Liquid can be used.

File: app/models/agents/telegram_agent.rb (hunk 1 above)
Comment (cantino) [inline app/models/agents/telegram_agent.rb]: Also consider `interpolated['parse_mode']` so Liquid can be used.

**Comment by cantino** [comment]
Cool!

**Comment by andrey-yantsen** [comment]
Updated

**Comment by cantino** [comment]
Thanks!

**Pull request merged**